src/extension.ts

```diff
@@ -434,23 +434,29 @@ function handleLogViaKubectlCli(podName: string) : Promise<string> {
 async function registerTutorialWithDidact(context: vscode.ExtensionContext) {
 	// call didact command to register tutorial
 	try {
-		// command ID: vscode.didact.register
-		const commandId = 'vscode.didact.register';
-
-		// then pass name, uri, and category
-		const tutorialName = 'Your First Integration';
-		const tutorialPath = path.join(context.extensionPath, './didact/camelk/first-integration.md');
-		const tutorialUri = vscode.Uri.parse(`file://${tutorialPath}`);
-		const tutorialCategory = 'Apache Camel K';
-
-		console.log('Tutorial URI registered: ' + tutorialUri.fsPath);
-
-		await vscode.commands.executeCommand(
-			commandId,
-			tutorialName, 
-			tutorialUri,
-			tutorialCategory);
-
+		// test to ensure didact is available 
+		const extensionId = 'redhat.vscode-didact';
+		const didactExt : any = vscode.extensions.getExtension(extensionId);
+
+		// if didact is available, register the new extension
```

Comment:
register the new extension? you mean register the tutorial in didact?

Reply:
Yes, good call. Will update the comment